lib/src/main/java/io/github/taz03/jia/InstagramClient.java

```diff
@@ -49,20 +56,40 @@ public final class InstagramClient {
      * @param password The instagram password corresponding to the username provided
      */
     public InstagramClient(String username, String password) {
-        this(username, password, null);
+        this.username = username;
+        this.password = password;
+    }
+
+    public InstagramClient(String user,String pass,String id,String agent){
```

Comment:
missing javadoc and wrong formatting

Reply:
That can be done later by you, the main POV is the code